test/lib/jdk/test/lib/security/OpensslArtifactFetcher.java

```diff
@@ -48,43 +49,39 @@ public class OpensslArtifactFetcher {
            and return that path, if download fails then return null.
      *
      * @return openssl binary path of the current version
-     * @throws SkippedException if a valid version of OpenSSL cannot be found
+     * @throws IOException if a valid version of OpenSSL cannot be found
```

Comment:
This still also throws `SkippedException`:
https://github.com/openjdk/jdk/blob/ae26c5a24e50584cfa3784c4b7e4cc84a738a530/test/lib/jdk/test/lib/security/OpensslArtifactFetcher.java#L83-L84

```suggestion
     * @throws SkippedException if a valid version of OpenSSL cannot be found
```